bridgesync/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   test-claimdata-l1:
     container_name: test-claimdata-l1
-    image: hermeznetwork/geth-zkevm-contracts:elderberry-fork.9-geth1.13.11
+    image: hermeznetwork/geth-zkevm-contracts:latest
```

Comment:
Do we want to always get the latest image of the contracts? Seems like it might cause issues with the test if something changes in the contracts.

Reply:
is this docker used nowadays!?